stdlib/public/core/Codable.swift.gyb

```diff
@@ -1180,7 +1180,7 @@ public enum EncodingError : Error {
   public var _userInfo: AnyObject? {
     // The error dictionary must be returned as an AnyObject. We can do this
     // only on platforms with bridging, unfortunately.
-    #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+    #if _runtime(_ObjC)
```

Comment:
We could actually remove this `#if` now, right? We should be able to return `[String : Any] as AnyObject` on all platforms, yeah?

Reply:
Oh, yeah :)